### PR TITLE
Fix type conversion issue of property Version for ManagementPartner API

### PR DESCRIPTION
**Pull request by neil-yechenwei**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by neil-yechenwei** [comment]
This PR is to fix type conversion bug (https://github.com/Azure/azure-rest-api-specs/issues/8178) of property Version of ManagementPartner API.

**Review by allenjzhang — APPROVED** [review]

File: specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/ManagementPartner.json [hunk 1]
```diff
@@ -291,7 +291,8 @@
           "description": "This is the object id."
         },
         "version": {
```
Comment (allenjzhang) [inline specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/ManagementPartner.json]: You must update the example JSON file to change the version from string "3" to int 3 in order for the validation check to pass

**Review by allenjzhang — CHANGES_REQUESTED** [review]

File: specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/ManagementPartner.json (hunk 1 above)
Comment (neil-yechenwei) [reply specification/managementpartner/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/ManagementPartner.json]: @allenjzhang , Thanks for your suggestion. I've fixed it.

**Review by jeffrey-ace — APPROVED** [review]

**Comment by neil-yechenwei** [comment]
@allenjzhang , it seems that the service owner "jeffrey-ace" has approved code. Could you help to merge the code? Thanks.

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**